src/language/dafnyLanguageClient.ts

```diff
@@ -41,16 +41,21 @@ function getMarkGhostStatementsArgument(): string {
 }
 
 export class DafnyLanguageClient extends LanguageClient {
+  private statusOutput: OutputChannel;
+
   // eslint-disable-next-line max-params
-  private constructor(id: string, name: string, serverOptions: ServerOptions, clientOptions: LanguageClientOptions, forceDebug?: boolean) {
+  private constructor(id: string, name: string, serverOptions: ServerOptions,
```

Comment:
Maybe start listing the constructor parameters on separate lines to have the closing parenthesis on its own line.